CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Community Code of Conduct
```

Comment:
Do you think since this is such a small doc we could add this as a section in the main README?

Reply:
we had to quickly add the code of conduct in order to continue with the cncf sandbox status.